canvas_modules/common-canvas/src/common-properties/form/ControlInfo.js

```diff
@@ -71,6 +71,12 @@ export class Control {
 		if (settings.charLimit) {
 			this.charLimit = settings.charLimit;
 		}
+		if (settings.toggleofflabel) {
```

Comment:
I don't think this make sense.  These need to be translatable strings and this wouldn't provide that.

Reply:
I think since we're reading these strings from parameter definition, these will be translatable strings. Same way label and description are translatable as it's coming from paramDef.